data/overview.yaml

```diff
@@ -59,106 +104,135 @@ Groups:
   representation theory for finite groups:
-    representations of abelian groups: ''[
+    representations of abelian groups: ''
+    dual groups: ''
     Maschke theorem: ''
     orthogonality of irreducible characters: ''
     Fourier transform for finite abelian groups: ''
     convolution: ''
+    class function over a group: ''
+    characters of a finite dimensional representation: ''
+    orthonormal basis of irreducible characters: ''
     examples for groups with a small cardinal: ''
 
 3. 
-Rings: 
-  Unitary rings:
+Rings:
+  ring:
+  Unitary ring:
   ring morphisms: 
   subrings: 
-  The ring Zn on the integers: 
+  The ring Z on the integers: 
+  Product of rings:
   Ideal of a commutative ring: 
   quotient rings: 
-  first ideals: 
+  prime ideals: 
   maximal ideals: 
   Chinese remainder theorem: 
-  Commutative ring algebra:
+  Algebra:
+    algebra over a commutative ring:
+    associative algebra over a commutative ring:
   Divisibility in commutative integral rings: 
   Irreducible elements: 
   invertible elements: 
-  first elements among them: 
-  Factorial rings: 
+  Coprimes elements: 
+  Unique factorization domain (UFD): 
   Greatest common divisor: 
-  least common multiple:
-  Factorability of A[X] when A is a factorial ring:
+  Least common multiple:
+  A[X] is a UFD when A is a UFD:
   Principal ring: 
   Bezouts theorem: 
   Euclidean rings: 
-  Euclids algorithm: 
-  Case of ring Z and K[X] algebra of polynomials on the field K: 
-  Irreducible polynomial: C
-  cyclotomic polynomials in Q[X]: 
+  Euclid's' algorithm: 
+  Z is an euclidean ring:
+  K[X] is an euclidean ring when K is a field:
+  Irreducible polynomial:
+  Cyclotomic polynomials in Q[X]: 
   Eisenstein's criterion:
   Congruence in Z: 
   Prime numbers: 
   Z/nZ and its invertible elements: 
-  Eurlers indicator function:
+  Euler's indicator function (phi):
 Polynomial:
   Polynomial algebra on one or several indeterminates over a commutative ring:
   Roots of a polynomial:
-  multiplicity: 
+  Multiplicity: 
   Relationship between the coefficients and the roots of a split polynomial: 
-  Newtonian sums: 
+  Newton's sums: 
   Derived polynomials: 
   Decomposition into sums of homogeneous polynomials: 
   Symmetric polynomials:
  Fields:
+  fields: 
   subfields: 
-  Characteristics: 
-  Frobenius morphisms: 
+  characteristic:
+  Frobenius' morphism: 
   field extensions: 
   integral ring fractional fields: 
   Field Q of rational numbers: 
   Field R of real numbers: 
   Field C of complex numbers: 
   D'Alembert-Gauss theorem: 
-  Algebraic elements transcendentals: 
+  Algebraic elements:
+  Transcendental elements: 
   Algebraic extensions: 
-  Closed algebraic fields: 
+  Algebraically closed fields: 
   Rupture fields:
   Splitting fields:
   Finite fields:
   Rational fraction fields with one indeterminate over the field: 
-  Decomposition into simple elements: 
-  Real and complex cases:
- 
-4. 
-Real Analysis:
```

Comment:
Where did this section go?

Reply:
It was "Single variable real analysis" from the start I think, in the agreg doc, I never found about this 4th part. I suspect a mistake

Reply:
ok